borgcron.sh

```diff
@@ -78,6 +78,13 @@ rm_lock() {
 	rm "$RUN_PID_DIR/BORG_$BACKUP_NAME.pid"
 }
 
+track_exitcode() {
+	EXITCODE_TEMP="$1"
```

Comment:
Variables not being constants, should be written in lower letters here, please. (also applies to `EXITCODE`)

Reply:
Wait, actually you do not need this temp variable anyway, so löeave them away, use "$1" directly, when needed.